urbanopt/geojson-modelica-translator
# 4G or 5G timeseries in Dymola

#### Any background context you want to provide?
The existing 4G models haven't been updated to use MBLv9.1. There are new partial buildings, partial ets', and other new features we can take advantage of.

#### What does this PR accomplish?
- Updates existing 4G models to take advantage of MBLv9 capabilities
- Updates code to allow user selection of district generation (fourth or fifth are supported currently)
- Creates 5G TimeSeries model
    - ETS is embedded in the building model
    - District model has fewer objects
    - New couplings to connect new ambient loop to new ports on the building
#### How should this be manually tested?
- Create a 5G model:
    - `poetry run pytest tests/model_connectors/test_time_series_5g.py -m 'not simulation and not compilation'`
- Open the model in Dymola (MSLv4, MBLv9)
    - Confirm no errors on compiling (no errors when loading the model in Dymola)
- Simulate the model
    - May desire to change simulation to 365 days for more comprehensive test
    - Confirm no errors when simulating
#### What are the relevant tickets?
Resolves #340 
Resolves #486 
Resolves #500 

#### Screenshots (if appropriate)
